### ISSUE TITLE
Keyboard navigation should enable you to add a feature image.

### ISSUE THREAD

**Issue by AfLosada** [AfLosada/Ghost_MISO#25]
**Describe the bug**
Keyboard navigation should enable you to add a feature image.
**To Reproduce**
1. Go to Pages
2. Try to navigate to Add feature image
4. You are unable to navigate correctly

**Expected behavior**

You should be able to selected through keyboard navigation an input field.

**Screenshots**

**Environment (please complete the following information):**
 - Device: Desktop
 - OS: Windows 11
 - Browser Chrome
 - Version 5.68.0

**Additional context**
